From cce31fb588c24c56848cb4ec28b10f2831232374 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Sat, 18 Jul 2020 02:09:18 +0200
Subject: [PATCH] Replace invalid GL_LINE with GL_LINES for glDrawArrays

GL_LINE applies to glPolygonMode, but glDrawArrays uses GL_LINES.

(cherry picked from commit bd241df6c5804875d879c4649915e3e83172ba3f)
---
 Rendering/ContextOpenGL2/vtkOpenGLContextDevice3D.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Rendering/ContextOpenGL2/vtkOpenGLContextDevice3D.cxx b/Rendering/ContextOpenGL2/vtkOpenGLContextDevice3D.cxx
index c55968d3b8..d24eeeda75 100644
--- a/Rendering/ContextOpenGL2/vtkOpenGLContextDevice3D.cxx
+++ b/Rendering/ContextOpenGL2/vtkOpenGLContextDevice3D.cxx
@@ -431,7 +431,7 @@ void vtkOpenGLContextDevice3D::DrawLines(
   this->BuildVBO(cbo, verts, n, colors, nc, nullptr);
   this->SetMatrices(cbo->Program);
 
-  glDrawArrays(GL_LINE, 0, n);
+  glDrawArrays(GL_LINES, 0, n);
 
   // free everything
   cbo->ReleaseGraphicsResources(this->RenderWindow);
-- 
2.27.0

